internal/corazarules/rule.go

```diff
@@ -76,3 +84,14 @@ func (r *RuleMetadata) Raw() string {
 func (r *RuleMetadata) SecMark() string {
 	return r.SecMark_
 }
+
+func (r *RuleMetadata) StrID() string {
+	if r.cachedStrID_ == "" {
```

Comment:
@jcchavezs is empty comparison as fast as a nil pointer comparison?

Reply:
Yeah, they both just compare a value to `0`